autode/opt/optimisers/rfo.py

```diff
@@ -108,7 +108,8 @@ def _take_step_within_trust_radius(
         if len(delta_s) == 0:  # No need to sanitise a null step
             return 0.0
 
-        new_coords = self._coords + factor * delta_s
+        step = factor * delta_s
+        new_coords = self._coords + step
```

Comment:
@t-young31 No, it's used on the next line `cartesian_delta = new_coords.to("cart") - self._coords.to("cart")`. This is basically required so that the final step can be taken by turning off the unconverged transform. RFO takes a large step sometimes, and the back transform does not converge. So using setting self._coords = new_coords has the risk of accepting unconverged coordinates.

Reply:
yep – sorry!